tests/installation/partitioning_lvm.pm

```diff
@@ -14,16 +14,24 @@ use base "y2logsstep";
 use testapi;
 
 sub run() {
-    my $self = shift;
+    my $self             = shift;
+    my $file_system_tags = [
+        qw/
+          partitioning-no-root-filesystem partitioning-encrypt-activated-existing
+          partitioning-encrypt-ignored-existing
+          /
+    ];
 
     if (get_var('ENCRYPT_ACTIVATE_EXISTING')) {
-        assert_screen [qw/partitioning-no-root-filesystem partitioning-encrypt-activated-existing/];
+        assert_screen $file_system_tags;
         if (match_has_tag('partitioning-no-root-filesystem')) {
             record_soft_failure 'bsc#989750';
         }
-        else {
-            return;
+        elsif (match_has_tag('partitioning-encrypt-ignored-existing')) {
+            record_soft_failure 'bsc#993247';
```

Comment:
I thought these were being collected, but it might just be old assets from a test run before I wiped the database on my local instance. It's not clear if a softfail will cause the post fail run hook to be called which collects logs in this case.